README.md

```diff
@@ -1,59 +1,89 @@
 
-Additionally you can configure favourites, set the unicode version, enable/disable categories and add custom emojis.  
-
-You can build this package from source of use the deb/rpm packages from https://github.com/alex1701c/EmojiRunner/releases.
+You can build this package from source or use the deb/rpm packages from [the releases page](https://github.com/alex1701c/EmojiRunner/releases).
 
 ### Required Dependencies
 
-Note: If the xdo.h file is not at compile time available the plugin will use the `xdotool` program to paste emojis, 
-but having the library installed is recommended.  
+**Note**: If the `xdo.h` file is not at compile time available the plugin will use the `xdotool` program to paste emojis, 
+but having the library installed is recommended.
+
+<details>
+<summary><b>Debian/Ubuntu</b></summary>
+
+```bash
+sudo apt install cmake extra-cmake-modules build-essential libkf5runner-dev libkf5textwidgets-dev qtdeclarative5-dev gettext libxdo-dev libkf5kcmutils-dev
+```
+
+</details>
+
+<details>
+<summary><b>OpenSUSE</b></summary>
+
+```bash
+sudo zypper install cmake extra-cmake-modules libQt5Widgets5 libQt5Core5 libqt5-qtlocation-devel ki18n-devel ktextwidgets-devel kservice-devel krunner-devel gettext-tools xdotool-devel kconfigwidgets-devel kcmutils-devel
+```
+
+</details>
+
+<details>
+<summary><b>Fedora</b></summary>
 
-Debian/Ubuntu:  
-`sudo apt install cmake extra-cmake-modules build-essential libkf5runner-dev libkf5textwidgets-dev qtdeclarative5-dev gettext libxdo-dev libkf5kcmutils-dev`
+```bash
+sudo dnf install cmake extra-cmake-modules kf5-ki18n-devel kf5-kservice-devel kf5-krunner-devel kf5-ktextwidgets-devel gettext xdotool kf5-kcmutils-devel
+```
 
-openSUSE:  
-`sudo zypper install cmake extra-cmake-modules libQt5Widgets5 libQt5Core5 libqt5-qtlocation-devel ki18n-devel
-ktextwidgets-devel kservice-devel krunner-devel gettext-tools xdotool-devel kconfigwidgets-devel kcmutils-devel`  
+</details>
 
-Fedora:  
-`sudo dnf install cmake extra-cmake-modules kf5-ki18n-devel kf5-kservice-devel kf5-krunner-devel kf5-ktextwidgets-devel gettext xdotool kf5-kcmutils-devel`  
+<details>
+<summary><b>Arch (Manjaro)</b></summary>
 
-Arch (Manjaro):  
-`sudo pacman -S cmake extra-cmake-modules xdotool kcmutils`  
+```bash
+sudo pacman -S cmake extra-cmake-modules xdotool kcmutils
+```
 *This xdotool package includes the xdo.h file*
 
+</details>
+
 ### Build instructions  
 
-The easiest way to install is:  
-`curl https://raw.githubusercontent.com/alex1701c/EmojiRunner/master/install.sh | bash`  
+<details>
+<summary>Option A: Easy oneliner method</summary>
+ 
+```bash
+curl https://raw.githubusercontent.com/alex1701c/EmojiRunner/master/install.sh | bash
+```
 
-You can also find the plugin at the AUR: https://aur.archlinux.org/packages/plasma5-runners-emoji/
+</details>
 
-Or you can do it manually:
+Option B: Get it from the [AUR](https://aur.archlinux.org/packages/plasma5-runners-emoji/)
 
-```
+<details>
```

Comment:
TBH I would not use the summary here, because the install instructions are pretty short and important information for readers